src/app/start-guide/what-is-markdown/page.md

```diff
@@ -1,112 +1,105 @@
 ---
-title: What and why is Markdown?
+title: What is Markdown and  why you should try it
 coverImage: '/images/what-is-markdown_cover.png'
 nextjs:
   metadata:
-    title: What and why is Markdown?
-    description: What is Markdown and why it is used
+    title: What is Markdown and  why you should try it
+    description: What is Markdown and  why you should try it
     openGraph:
       images:
         [
           'https://inkdrop-user-guide.vercel.app/images/what-is-markdown_cover.png',
         ]
 ---
 
-After installing Inkdrop, you may notice that the example notes in the app are written in a format called Markdown. If you're new to this, don't worry. We're here to introduce you to the world of Markdown and explain why it's such a favorite among developers and tech enthusiasts.{% .lead %}
+After installing Inkdrop, you may notice that the examples in the app contain symbols like `#` or `**`. These are Markdown syntax — a simple yet powerful way to format text. If you're new to this, no worries. This page introduces you to the world of Markdown and explains why it's such a favorite among developers and tech enthusiasts .{% .lead %}
 
-## Inkdrop is like a drafting table for programmers
+## What is Markdown? 🤔
 
-Imagine you're an architect. You have ideas brimming in your mind, designs waiting to take shape.
-Now, you could sketch these on any piece of paper, but you choose a drafting table.
-Why? Because it's tailored for your work.
-It has the right tools and features that make designing more efficient, precise, and effective.
-This is similar to why you should choose Markdown for writing your notes, instead of sticking to plain text.
-And, Inkdrop is the best drafting table for taking tech notes in Markdown.
+Markdown is a lightweight markup language that uses special syntax to format text. It's easy to learn and use. You don't need HTML or other cumbersome markup to make **bolds**, *italics*, [links](https://www.craftz.dog/), and other formatting must-haves. It's also a great way to take notes, write documentation, and create content for the web.
```

Comment:
Can you tell me the reason behind this change? I like using the metaphor of 'drafting table'. It has been working well to explain to non-techie people, so should beginners.

Reply:
Sure, with AI tools many docs start to resemble prose. In my suggestion I tried to be more straightforward. If you want to use metaphors, I revert it back.

P.S. because I changed the structure of the document, git displayed this change. But it fact, 'drafting table' is used further (line 33)

> Imagine you're an architect. You have ideas brimming in your mind, designs waiting to take shape.
Now, you could sketch these on any piece of paper, but you choose a drafting table.
Why? Because it's tailored for your work.
It has the right tools and features for productive work.
With Markdown, you don't write notes, you craft them. And, Inkdrop is the best tool for crafting.

Reply:
It is a metaphor I use when explaining to non-programmers like my parents. and, this page is directed at beginners who even don't understand what 'lightweight markup language' means.

Yeah, the 'Why should I try it?' looks nice! I should have been more careful to check it. Thanks.